eland/ml/ml_model.py

```diff
@@ -380,6 +376,120 @@ def import_model(
         >>> # Delete model from Elasticsearch
         >>> es_model.delete_model()
         """
+
+        return cls._import_model(
+            es_client=es_client,
+            model_id=model_id,
+            model=model,
+            feature_names=feature_names,
+            classification_labels=classification_labels,
+            classification_weights=classification_weights,
+            es_if_exists=es_if_exists,
+        )
+
+    @classmethod
+    def import_ltr_model(
```

Comment:
Question, can this be used to import models for any use cases besides LTR?

Reply:
Nope. For other use case you have to use the general purpose `import_model` method.